partials/author-bio-social-links.php

```diff
@@ -23,25 +23,25 @@
 <ul class="social-links">
 	<?php if ( $fb = $author_obj->fb ) { ?>
 		<li class="facebook">
-			<a href="https://facebook.com/<?php echo largo_fb_url_to_username(esc_url( $fb )); ?>" title="<?php echo esc_attr( $author_obj->display_name ); ?> on Facebook" rel="me"><i class="icon-facebook"></i></a>
+			<a href="https://facebook.com/<?php echo largo_fb_url_to_username(esc_url( $fb )); ?>" title="Follow <?php echo esc_attr( $author_obj->display_name ); ?> on Facebook" rel="me"><i class="icon-facebook"></i></a>
 		</li>
 	<?php } ?>
 
 	<?php if ( $twitter = $author_obj->twitter ) { ?>
 		<li class="twitter">
-			<a href="https://twitter.com/<?php echo largo_twitter_url_to_username ( $twitter ); ?>"><i class="icon-twitter"></i></a>
+			<a href="https://twitter.com/<?php echo largo_twitter_url_to_username ( $twitter ); ?>" title="Follow <?php echo largo_twitter_url_to_username ( $twitter ); ?> on Twitter" ><i class="icon-twitter"></i></a>
 		</li>
 	<?php } ?>
 
 	<?php if ( $email && $show_email !== 'off' ) { ?>
 		<li class="email">
-			<a class="email" href="mailto:<?php echo esc_attr( $email ); ?>" title="e-mail <?php echo esc_attr( $author_obj->display_name ); ?>"><i class="icon-mail"></i></a>
+			<a class="email" href="mailto:<?php echo esc_attr( $email ); ?>" title="E-mail <?php echo esc_attr( $author_obj->display_name ); ?>"><i class="icon-mail"></i></a>
```

Comment:
While we're making changes to this, would you mind removing the `-` in `E-mail`? The dash is somewhat dated.

Reply:
Hi @benlk , comment is addressed